lambda-search/update_config.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+source keys.sh
+
+case $EMBEDDING in
+("COHERE"|"OPEN_AI"))
```

Comment:
Ends up with:

```
deploy_lambda.sh: line 8: syntax error near unexpected token `)'
deploy_lambda.sh: line 8: `("COHERE"|"OPEN_AI"))'
```

Reply:
Argh, I added this stuff to make the scripts embedding-agnostic, but that's no longer needed. Removing.